core/connection.js

```diff
@@ -675,3 +697,103 @@ Blockly.Connection.prototype.toString = function() {
   }
   return msg + block.toDevString();
 };
+
+/**
+ * Returns the state of the shadowDom_ and shadowState_ properties, then
+ * temporarily sets those properties to null so no shadow respawns.
+ * @return {{shadowDom: ?Element, shadowState: ?Object}} The state of both the
+ *     shadowDom_ and shadowState_ properties.
+ */
```

Comment:
This function and many of the ones below it look like they're intended to be private/package, but they don't have access modifiers.